### PR TITLE
Feature/sie 57 account page

### PR DESCRIPTION
**Pull request by cmutuc25**
[Account Page Ticket](https://cs6510.atlassian.net/browse/SIE-57)

Account Page used by the users to edit their info and delete their accounts
Added some css to the `Form` and `FormItem` specific to this page.
Added `AccountPage` and `AccountPageInfo` which make up the full page.

Changed the `getCurrentUsers` to run the callback that is passed in even if no user is found.

### CODE REVIEW

**Comment by cmutuc25** [comment]
Dont merge this yet

File: .prettierrc.js
Comment (alecherryy) [inline .prettierrc.js:12]: SWEET. I have been meaning to do this.

**Comment by cmutuc25** [comment]
This is ready to go now, took some liberty with the Sketch image, people should take a look at some of the tests to see how some mocking examples

**Review by harmanwho — APPROVED** [review]

**Pull request merged**
